dimod/discrete/cydiscrete_quadratic_model.pyx

```diff
@@ -18,6 +18,7 @@ cimport cython
 
 from cython.operator cimport preincrement as inc, dereference as deref
 from cython.operator cimport preincrement as preinc  # for when it matters
+from libcpp cimport bool
```

Comment:
no harm using `bool`, but `bint` does not need to be cimported.

Reply:
I came across `bint` too and for the sake of import reduction, I'll use it